src/Microsoft.AspNet.Razor/Parser/CSharpCodeParser.Statements.cs

```diff
@@ -13,6 +13,9 @@ namespace Microsoft.AspNet.Razor.Parser
 {
     public partial class CSharpCodeParser
     {
+        private static readonly Func<CSharpSymbol, bool> IsValidStatementSpacingSymbol =
+            IsSpacingToken(includeNewLines: true, includeComments: true);
```

Comment:
Seems that `IsSpacingToken` is called all over with different variations of the parameters. I opted to not go with the `static method` approach so the `Func<CSharpSymbol, bool>` is only ever created once.